src/css/original_assets/feedback.css

```diff
@@ -2,6 +2,10 @@
   min-height: 37rem;
   margin-bottom: 17.25rem;
 }
+.work-style {
+  z-index: -100 !important;
```

Comment:
too much. it is enough to have z-index 1 on the testimonials and -1 for the workstyle ..